bitmap-plasma/app/build.gradle

```diff
@@ -19,7 +19,7 @@ model {
 
     android.buildTypes {
         release {
-            isMinifyEnabled = false
+            minifyEnabled = false
```

Comment:
isJniDebuggable is now a NDK buildType options and renamed to debuggable.  You can specify it with something like this:
android.buildType {
    release {
        ndk {
            debuggable = true
        }
    }
}